ccstake/src/cmds/batchDelegate.ts

```diff
@@ -0,0 +1,627 @@
+import { SDK } from "codechain-sdk";
+import {
+    PlatformAddress,
+    SignedTransaction,
+    U64
+} from "codechain-sdk/lib/core/classes";
+import * as stake from "codechain-stakeholder-sdk";
+import * as fs from "fs";
+import * as util from "util";
+import * as yargs from "yargs";
+
+import prompts = require("prompts");
+import { GlobalParams } from "..";
+import { summarize, Summary } from "../summerizer";
+import { asyncHandler, createTable, prologue } from "../util";
+
+interface BatchDelegateParams extends GlobalParams {
+    "distribution-file": string;
+    "password-path": string;
+    "dry-run": boolean;
+}
+
+export const module: yargs.CommandModule<GlobalParams, BatchDelegateParams> = {
+    command: "batch-delegate <distribution-file>",
+    describe: "Batch manage delegations through distribution file",
+    builder(args) {
+        return args
+            .positional("distribution-file", {
+                desc: "File to describe the distribution of delegations",
+                type: "string",
+                normalize: true
+            })
+            .demandOption("distribution-file")
```

Comment:
What is the difference from using `demandOption` method and using `demand` field in option?

Reply:
No difference. I prefer `.demand` option field, however `.positional` doesn't have it like `.option` does.